>>> 'Pkg.add("KCores")' log
INFO: Cloning cache of KCores from https://github.com/johnybx/KCores.jl.git
INFO: Installing BinDeps v0.6.0
INFO: Installing Conda v0.5.3
INFO: Installing DataStructures v0.6.0
INFO: Installing Graphs v0.7.1
INFO: Installing KCores v0.0.3
INFO: Installing MacroTools v0.3.7
INFO: Installing PyCall v1.14.0
INFO: Installing SHA v0.3.3
INFO: Installing URIParser v0.1.8
INFO: Building Conda
INFO: Building PyCall
INFO: Using the Python distribution in the Conda package by default.
To use a different Python version, set ENV["PYTHON"]="pythoncommand" and re-run Pkg.build("PyCall").
Fetching package metadata .....
CondaHTTPError: HTTP 504 GATEWAY TIME-OUT for url <https://conda.anaconda.org/conda-forge/linux-64/repodata.json>
Elapsed: 00:59.131844
CF-RAY: 37d601f468b05a80-BOS

An remote server error occurred when trying to retrieve this URL.

A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to
fulfill a valid request.  The problem may be spurious, and will resolve itself if you
try your request again.  If the problem persists, consider notifying the maintainer
of the remote server.


===============================[ ERROR: PyCall ]================================

LoadError: failed process: Process(setenv(`/home/vagrant/.julia/v0.5/Conda/deps/usr/bin/conda install -y numpy`,String["PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/","PWD=/vagrant/0.5AK","SHELL=/bin/bash","SSH_CONNECTION=10.0.2.2 35285 10.0.2.15 22","LD_LIBRARY_PATH=:/usr/local/lib/","CONDARC=/home/vagrant/.julia/v0.5/Conda/deps/usr/condarc-julia.yml","MAIL=/var/mail/vagrant","LANG=en_US.UTF-8","SHLVL=3","LOGNAME=vagrant","XDG_RUNTIME_DIR=/run/user/1000","XDG_SESSION_ID=3","CONDA_PREFIX=/home/vagrant/.julia/v0.5/Conda/deps/usr","OLDPWD=/home/vagrant","SSH_CLIENT=10.0.2.2 35285 22","_=/usr/bin/timeout","CONDA_DEFAULT_ENV=/home/vagrant/.julia/v0.5/Conda/deps/usr","USER=vagrant","TERM=vt100","HOME=/home/vagrant","OPENBLAS_MAIN_FREE=1","JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64"]), ProcessExited(1)) [1]
while loading /home/vagrant/.julia/v0.5/PyCall/deps/build.jl, in expression starting on line 258

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: PyCall had build errors.

 - packages with build errors remain installed in /home/vagrant/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("PyCall")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of KCores
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("KCores")' log
Julia Version 0.5.2
Commit f4c6c9d (2017-05-06 16:34 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-121-generic #170-Ubuntu SMP Wed Jun 14 09:04:33 UTC 2017 x86_64 x86_64
Memory: 2.9392738342285156 GB (905.015625 MB free)
Uptime: 50310.0 sec
Load Avg:  0.3251953125  0.689453125  0.853515625
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3501 MHz    3164016 s         71 s     251642 s     929539 s         20 s
#2  3501 MHz     908062 s       5995 s     111746 s    3878566 s          0 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1 (ORCJIT, haswell)
Environment:
  TERM = vt100
  LD_LIBRARY_PATH = :/usr/local/lib/
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-8-openjdk-amd64
  HOME = /home/vagrant

Package Directory: /home/vagrant/.julia/v0.5
2 required packages:
 - JSON                          0.12.0
 - KCores                        0.0.3
10 additional packages:
 - BinDeps                       0.6.0
 - Compat                        0.26.0
 - Conda                         0.5.3
 - DataStructures                0.6.0
 - Graphs                        0.7.1
 - MacroTools                    0.3.7
 - PyCall                        1.14.0
 - SHA                           0.3.3
 - URIParser                     0.1.8
 - tmp                           0.0.0-             non-repo (unregistered)
INFO: Testing KCores
ERROR: LoadError: LoadError: PyCall not properly installed. Please run Pkg.build("PyCall")
 in macro expansion; at /home/vagrant/.julia/v0.5/PyCall/src/startup.jl:45 [inlined]
 in anonymous at ./<missing>:?
 in include_from_node1(::String) at ./loading.jl:488 (repeats 2 times)
 in macro expansion; at ./none:2 [inlined]
 in anonymous at ./<missing>:?
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:242
 in _start() at ./client.jl:321
while loading /home/vagrant/.julia/v0.5/PyCall/src/startup.jl, in expression starting on line 42
while loading /home/vagrant/.julia/v0.5/PyCall/src/PyCall.jl, in expression starting on line 39
ERROR: LoadError: LoadError: Failed to precompile PyCall to /home/vagrant/.julia/lib/v0.5/PyCall.ji.
 in compilecache(::String) at ./loading.jl:593
 in require(::Symbol) at ./loading.jl:393
 in include_from_node1(::String) at ./loading.jl:488
 in macro expansion; at /home/vagrant/.julia/v0.5/KCores/test/runtests.jl:3 [inlined]
 in anonymous at ./<missing>:?
 in include_from_node1(::String) at ./loading.jl:488
 in process_options(::Base.JLOptions) at ./client.jl:265
 in _start() at ./client.jl:321
while loading /home/vagrant/.julia/v0.5/KCores/test/pytests.jl, in expression starting on line 2
while loading /home/vagrant/.julia/v0.5/KCores/test/runtests.jl, in expression starting on line 2
===============================[ ERROR: KCores ]================================

failed process: Process(`/home/vagrant/julia/bin/julia -Cx86-64 -J/home/vagrant/julia/lib/julia/sys.so --compile=yes --depwarn=yes --check-bounds=yes --code-coverage=none --color=no --compilecache=yes /home/vagrant/.julia/v0.5/KCores/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: KCores had test errors
 in #test#61(::Bool, ::Function, ::Array{AbstractString,1}) at ./pkg/entry.jl:748
 in (::Base.Pkg.Entry.#kw##test)(::Array{Any,1}, ::Base.Pkg.Entry.#test, ::Array{AbstractString,1}) at ./<missing>:0
 in (::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}})() at ./pkg/dir.jl:31
 in cd(::Base.Pkg.Dir.##2#3{Array{Any,1},Base.Pkg.Entry.#test,Tuple{Array{AbstractString,1}}}, ::String) at ./file.jl:59
 in #cd#1(::Array{Any,1}, ::Function, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./pkg/dir.jl:31
 in (::Base.Pkg.Dir.#kw##cd)(::Array{Any,1}, ::Base.Pkg.Dir.#cd, ::Function, ::Array{AbstractString,1}, ::Vararg{Array{AbstractString,1},N}) at ./<missing>:0
 in #test#3(::Bool, ::Function, ::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
 in test(::String, ::Vararg{String,N}) at ./pkg/pkg.jl:258
 in eval(::Module, ::Any) at ./boot.jl:234
 in process_options(::Base.JLOptions) at ./client.jl:242
 in _start() at ./client.jl:321

>>> End of log
